CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ### Features
 
+- `[jest-cli]` Only set error process error codes when they are non-zero ([#7363](https://github.com/facebook/jest/pull/7363))
```

Comment:
I believe this depends on your expectation. 

If on error you expect it to exit with code 1, that still happens, so non-breaking.

If you expect things to exit code 0 on success no matter what, then yes. However I would say this is a bug fix for the latter case.

Reply:
Makes sense - for me this is a change in functionality more than it is a bug fix but I see where you're coming from

Either way that opinion lands, this seems pretty likely to break people (especially in CI workflows) so for their benefit (and for the benefit of the issue tracker) it may be good to label this as breaking

Reply:
A bug fix can be a breaking change 🙂 So I'm +1 to marking it as breaking in order tohighlightt it a bit if people run into it